blocks/extras/extensions/plot.py

```diff
@@ -6,6 +6,7 @@
 try:
     from bokeh.plotting import (curdoc, cursession, figure, output_server,
                                 push, show)
+    from bokeh.util.platform import is_notebook
```

Comment:
Which bokeh version is necessary for this to work? I have 0.8.1 and it seems insufficient.